src/test/java/it/mulders/mcs/search/SearchCommandHandlerTest.java

```diff
@@ -15,7 +16,7 @@
 
 @DisplayNameGeneration(DisplayNameGenerator.ReplaceUnderscores.class)
 class SearchCommandHandlerTest implements WithAssertions {
-    private final OutputPrinter outputPrinter = mock(OutputPrinter.class);
+    private final DelegatingOutputPrinter outputPrinter = mock(DelegatingOutputPrinter.class);
```

Comment:
Now that the `DelegatingOutputPrinter` is immutable again, could we rollback the declaration of this instance variable to type `OutputPrinter`?

Reply:
Switched back to `OutputPrinter`. Thank you for make me aware of.